css/datagrid/datagrid.css

```diff
@@ -256,42 +199,122 @@
+    color: var(--highcharts-neutral-color-100);
+}
+
+/* Table */
+.highcharts-datagrid-table {
+    color: var(--highcharts-neutral-color-100);
+    background-color: var(--highcharts-background-color);
+    border: 1px solid var(--highcharts-neutral-color-20);
+}
+
+.highcharts-datagrid-caption {
+    padding: 5px;
+}
+
+/* Table header */
+.highcharts-datagrid-table thead {
+    background-color: var(--highcharts-neutral-color-5);
+    border-bottom: 1px solid var(--highcharts-neutral-color-20);
+}
+
+.highcharts-datagrid-table thead tr {
+    height: 32px;
+    border-top: 1px solid var(--highcharts-neutral-color-20);
+}
+
+.highcharts-datagrid-table thead tr:first-child {
+    border-top: none;
+}
+
+.highcharts-datagrid-table thead th {
+    border-left: 1px solid var(--highcharts-neutral-color-20);
+}
+
+.highcharts-datagrid-table thead th.highcharts-datagrid-hovered-column {
+    background-color: var(--highcharts-highlight-color-10);
 }
 
 .highcharts-datagrid-header-cell-content::after {
-    right: 10px;
-    top: calc(50% - 10px);
-    bottom: 50%;
     color: var(--highcharts-neutral-color-60);
-    position: absolute;
-    display: inline-block;
-    width: 12px;
-    height: 12px;
 }
 
-/* stylelint-disable-next-line max-line-length */
-.highcharts-datagrid-column-sortable.highcharts-datagrid-column-sorted-asc .highcharts-datagrid-header-cell-content::after {
-    content: "▲";
+.highcharts-datagrid-table td:first-child,
+.highcharts-datagrid-table thead th.highcharts-datagrid-column-first,
+.highcharts-datagrid-table thead tr:first-child th:first-child {
+    border-left: none;
 }
 
-/* stylelint-disable-next-line max-line-length */
-.highcharts-datagrid-column-sortable.highcharts-datagrid-column-sorted-desc .highcharts-datagrid-header-cell-content::after {
-    content: "▼";
+/* Table body */
+.highcharts-datagrid-table tbody tr {
+    background: var(--highcharts-background-color);
+    border-bottom: 1px solid var(--highcharts-neutral-color-20);
+}
+
+.highcharts-datagrid-table tbody tr.highcharts-datagrid-row.highcharts-datagrid-row-odd {
+    background-color: var(--highcharts-neutral-color-3);
+}
+
+.highcharts-datagrid-table tbody td {
+    border-left: 1px solid var(--highcharts-neutral-color-20);
+    padding: 0 4px;
 }
+
+.highcharts-datagrid-table tbody tr.highcharts-datagrid-hovered-row,
+.highcharts-datagrid-table tbody tr.highcharts-datagrid-row.highcharts-datagrid-row-odd.highcharts-datagrid-hovered-row {
+    background-color: var(--highcharts-highlight-color-10);
+}
+
+.highcharts-datagrid-table tbody tr.highcharts-datagrid-hovered-row td.highcharts-datagrid-hovered-column {
+    background-color: var(--highcharts-highlight-color-10);
+}
+
+.highcharts-datagrid-table tbody td .highcharts-datagrid-focused-cell input,
+.highcharts-datagrid-table tbody td .highcharts-datagrid-focused-cell input:focus {
+    border: 1px solid #0000ed;
+    background-color: #b7d4ff;
+}
+
+/* End Default Theme */
```

Comment:
Isn't the CSS linter blocking your commit? 😄 It seems there should be an empty newline at the end of the file.